web/book/src/SUMMARY.md

```diff
@@ -55,6 +55,13 @@
   - [F-strings](./language-features/f-strings.md)
   - [Target & Version](./language-features/target.md)
 
+- [Compiler](./compiler/README.md)
+
+  - [Reference implementation](./compiler/rust/README.md)
+    - [CLI](./compiler/rust/cli.md)
+  - [Compiler features](./compiler/compiler-features/README.md)
+    - [Target & Version](./compiler/compiler-features/target.md)
+
 - [Bindings](./bindings/README.md)
```

Comment:
Bindings are technically "Language bindings to the Rust prql-compiler library", so they should maybe be under "Compiler" section too.

Or maybe, "Bindings" section should be titled "Compiler" and contain a page about the CLI and a page about "Target & version"